kineticTypo/kineticTypo1/Tile.pde

```diff
@@ -1,102 +1,100 @@
 class Tile {
-
- float tx;
- float ty;
   
- int tilesX; 
- int tilesY;
+  // Since we configure this at the beginning we can make this
+  // variable a final static variable, this means we know 
+  // that this variable will never change
+  private final static int fraqX = 4; 
+  private final static int fraqY = 6;
+  private final static int tolleranceX = 30;
```

Comment:
Using `private final static` clearly states that this is a configuration of the Tiles that will never change during runtime.